c++/src/kj/compat/url-test.c++

```diff
@@ -36,6 +36,11 @@ Url parseAndCheck(kj::StringPtr originalText, kj::StringPtr expectedRestringifie
   return url;
 }
 
+static constexpr Url::Options NO_DECODE {
+  /* .percentDecode = */ false,
+  // TODO(cleanup): Use designated initializer once MSVC supports them.
```

Comment:
Designated initializers are not standardized for C++, only C. Some compilers support them in C++ but we shouldn't rely on that until they're actually standardized (maybe C++20, it looks like).